features/support/publishing_cuke_helpers.rb

```diff
@@ -27,6 +27,8 @@ def submit_publisher
     txt = find("#publisher #status_message_text").value
     find("#publisher .btn-primary").click
     # wait for the content to appear
+    expect(find("#publisher")).not_to have_css(".btn-primary")
```

Comment:
This is still failing:
`expected not to find css ".btn-primary", found 1 match: "Share". Also found "", which matched the selector but not all filters. (RSpec::Expectations::ExpectationNotMetError)`